ReactiveCocoa/Swift/Signal.swift

```diff
@@ -331,6 +331,21 @@ extension SignalType {
 	}
 }
 
+extension SignalType {
+	/// Merges the given signals into a single `Signal` that will emit all values
+	/// from each of them, and complete when all of them have completed.
+	public static func mergeSignals(signals: [Signal<Value, Error>]) -> Signal<Value, Error> {
```

Comment:
I agree about `merge`. Also, could this take a `SequenceType` containing `Signal`s instead of an array?